app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,12 @@ html dir="ltr" lang="en-GB" class="govuk-template"
     = tag.meta name: "viewport", content: "width=device-width, initial-scale=1"
     = tag.meta property: "og:image", content: asset_path("images/govuk-opengraph-image.png")
     = tag.meta name: "theme-color", content: "#0b0c0c"
-    = tag.meta name: "robots", content: "noindex"
+    ruby:
+      index_prod = FeatureFlags.enabled?(:index_production, without_session_data: true) ? "" : "noindex"
```

Comment:
If we do want Google to index the site, we'll have a directive that looks like: `<meta name="robots" content="">`. I'm not sure it's [strictly valid](https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#directives) to have a robots directive with blank content. To be on the safe side, maybe we should have it so that the "content" value if we do want indexing is `"all"` rather than `""`.